sdk/storage/azblob/zc_leaseClient.go

```diff
@@ -0,0 +1,168 @@
+package azblob
```

Comment:
Just a general note: I've noticed that this and many other files need the license header. This can be done in a separate PR, but just an FYI.

Reply:
Hey @catalinaperalta ,

Thanks for your input.  I noticed the same today when I was checking the logs [here](https://dev.azure.com/azure-sdk/public/_build/results?buildId=748504&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=caefc57e-610d-58f0-579e-b7adfd9fa150). I'll talk to my peers and get this done.

Reply:
Example license header: 
```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```
I believe this is the latest license header we're using.